plugins/vestige/vestige.cpp

```diff
@@ -60,8 +60,6 @@
 
 #include "embed.h"
 
-
-
```

Comment:
As a reminder: I recommend you reverting irrelevant changes in new lines and whitespace after finishing this PR.